src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+const Calculator = () => {
+  const [state] = useState({ total: 0 });
+  const { total } = state;
+
+  return (
```

Comment:
- [ ] Kindly convert the Calculator component to be class-based component instead of a function-based component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).).